example/CMakeLists.txt

```diff
@@ -29,7 +29,13 @@ set(CUSTOM_PROXY_SOURCES
 
 # Add libmexclass as an external project.
 include(ExternalProject)
-ExternalProject_Add(
+# Clients must use the libmexclass/cpp/libmexclassInstall.cmake to complete the installation steps and package the binaries.
```

Comment:
Suggested wording: "Clients must use the libmexclass/cpp/libmexclassInstall.cmake..." -> "Clients must use libmexclass/cpp/libmexclassInstall.cmake..."

Reply:
Fixed in commit 309a468.